pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -184,10 +207,13 @@ func buildContainerEnvs(dk *dynakube.DynaKube) []corev1.EnvVar {
 		{Name: envServerUrl, Value: buildActiveGateServiceName(dk) + "." + dk.Namespace + ".svc.cluster.local:443"},
 		{Name: envEecTokenPath, Value: eecTokenMountPath + "/" + eecFile},
 		{Name: envEecIngestPort, Value: strconv.Itoa(int(collectorPort))},
-		{Name: envExtensionsConfPathName, Value: envExtensionsConfPath},
 		{Name: envExtensionsModuleExecPathName, Value: envExtensionsModuleExecPath},
 		{Name: envDsInstallDirName, Value: envDsInstallDir},
 		{Name: envK8sClusterId, Value: dk.Status.KubeSystemUUID},
+		{Name: envK8sExtServiceUrl, Value: serviceAccountName},
+		{Name: envHttpsCertPathPem, Value: path.Join(httpsCertMountPath, httpsCertFile)},
```

Comment:
yeah, I also thought about it, but then I switched to just joining strings into constants so we can reuse them instead of using the `path.Join` in every occurrence.